Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -1242,12 +1239,8 @@ Path InspectorOverlay::drawElementTitle(GraphicsContext& context, Node& node, co
         }
     }
 
-    auto* localMainFrame = dynamicDowncast<LocalFrame>(m_page.mainFrame());
-    if (!localMainFrame)
-        return { };
-
-    auto* pageView = localMainFrame->view();
-
+    Ref mainFrame = m_page.mainFrame();
+    RefPtr pageView = mainFrame->virtualView();
```

Comment:
At some point we should rename view to localView and remoteView and virtualView to view.